test/smtcallback.ts

```diff
@@ -25,7 +25,7 @@ function collectErrors (solOutput) {
   return errors;
 }
 
-function expectErrors (expectations, errors, ignoreCex) {
+function expectErrors (expectations, errors, ignoreCex?: any) {
```

Comment:
I think `ignoreCex` may be better if required as a flag, but it is up to @leonardoalt.

Reply:
this has to be nullable and any since its not being used in the test cases.

```
test/smtcallback.ts:122:13 - error TS2554: Expected 3 arguments, but got 2.

122       st.ok(expectErrors(errors, test.expectations));
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  test/smtcallback.ts:28:46
    28 function expectErrors (expectations, errors, ignoreCex) {
                                                    ~~~~~~~~~
    An argument for 'ignoreCex' was not provided.


Found 1 error.
```

Reply:
I'm personally fine with either. Could also make it mandatory

Reply:
Oh, this is in tests. I'd just make it mandatory in tests and be done with it.